x-pack/metricbeat/module/aws/rds/_meta/fields.yml

```diff
@@ -0,0 +1,218 @@
+- name: rds
+  type: group
+  description: >
+    `rds` contains the metrics that were scraped from AWS CloudWatch which contains monitoring metrics sent by AWS RDS.
+  release: beta
+  fields:
+    - name: cpu.total.pct
+      type: scaled_float
+      format: percent
+      description: >
+        The percentage of CPU utilization.
+    - name: cpu.credit_usage
+      type: long
+      description: >
+        The number of CPU credits spent by the instance for CPU utilization.
+    - name: cpu.credit_balance
+      type: long
+      description: >
+        The number of earned CPU credits that an instance has accrued since it was launched or started.
+    - name: database_connections
+      type: long
+      description: >
+        The number of database connections in use.
+    - name: db_instance.arn
+      type: keyword
+      description: >
+        Amazon Resource Name(ARN) for each rds.
+    - name: db_instance.class
+      type: keyword
+      description: >
+        Contains the name of the compute and memory capacity class of the DB instance.
+    - name: db_instance.identifier
+      type: keyword
+      description: >
+        Contains a user-supplied database identifier. This identifier is the unique key that identifies a DB instance.
+    - name: db_instance.status
+      type: keyword
+      description: >
+        Specifies the current state of this database.
+    - name: disk_queue_depth
+      type: long
+      description: >
+        The number of outstanding IOs (read/write requests) waiting to access the disk.
+    - name: failed_sql_server_agent_jobs
+      type: long
+      description: >
+        The number of failed SQL Server Agent jobs during the last minute.
+    - name: freeable_memory.bytes
+      type: long
+      format: bytes
+      description: >
+        The amount of available random access memory.
+    - name: free_storage.bytes
+      type: long
+      format: bytes
+      description: >
+        The amount of available storage space.
+    - name: maximum_used_transaction_ids
+      type: long
+      description: >
+        The maximum transaction ID that has been used. Applies to PostgreSQL.
+    - name: oldest_replication_slot_lag.mb
+      type: long
+      description: >
+        The lagging size of the replica lagging the most in terms of WAL data received. Applies to PostgreSQL.
```

Comment:
I don't see these in the `data.json` file, does that mean it's not a postgres?

Reply:
Yes, the data.json file is generated with mysql I believe.